server/sonar-process/src/main/java/org/sonar/process/ProcessProperties.java

```diff
@@ -48,15 +49,16 @@
   public static final String PATH_TEMP = "sonar.path.temp";
   public static final String PATH_WEB = "sonar.path.web";
 
+  public static final String SEARCH_CLUSTER_NAME = "sonar.search.clusterName";
```

Comment:
This It's already done for the NODE_NAME, I missing the reason we don't do it for the cluster name